### PR TITLE
feat: Implement Inline Position Timeline

### PR DESCRIPTION
**Pull request by tobiasny**

### CODE REVIEW

File: src/components/org/PositionTimeline/InlinePositionTimeline/InlineSplit/components/MicroSplits.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+export const InlineSplits: FC<InlineSplitsProps> = ({
+    selectedSplit,
+    rotationColumns,
+    computePosition,
+    size,
+}) => {
+    return (
+        <>
+            {Object.values(rotationColumns).map((col) => (
+                <MicroSplit
```
Comment (eskil0312) [inline src/components/org/PositionTimeline/InlinePositionTimeline/InlineSplit/components/MicroSplits.tsx:21]: Rename

File: src/components/org/PositionTimeline/InlinePositionTimeline/InlineMarker/MicroMarkers.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import { FC, useMemo } from 'react';
```
[the remark this answers is not in the document]
Comment (eskil0312) [reply src/components/org/PositionTimeline/InlinePositionTimeline/InlineMarker/MicroMarkers.tsx:1]: Should MicroMarkers(plural) be inside be inside InlineMarker(singular)? Should be the opposite or in separate folders in my option

File: src/components/org/PositionTimeline/InlinePositionTimeline/InlineMarker/MicroMarkers.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+    const markerData: InlineMarkerData[] = useMemo(() => {
+        return generateMarkerData(sortedColumns);
+    }, [sortedColumns]);
+
+    return (
+        <>
+            {markerData.map((data) => (
+                <InlineMarker
+                    key={`micro-marker-${data.date.getTime()}`}
```
Comment (eskil0312) [inline src/components/org/PositionTimeline/InlinePositionTimeline/InlineMarker/MicroMarkers.tsx:35]: change to inline marker key

File: src/components/org/PositionTimeline/InlinePositionTimeline/InlineMarker/MicroMarkers.tsx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+                    selected={selectedSplit}
+                    date={data.date}
+                    linked={data.linked}
+                    computePosition={computePosition}
+                    size={size}
+                />
+            ))}
+            {selectedDate && (
+                <InlineMarker
+                    key={`micro-marker-${selectedDate.getTime()}`}
```
Comment (eskil0312) [inline src/components/org/PositionTimeline/InlinePositionTimeline/InlineMarker/MicroMarkers.tsx:45]: inline marker key

File: src/components/org/PositionTimeline/InlinePositionTimeline/InlineMarker/styles.ts [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+                ? theme.colors.interactive.primary__resting.getVariable('color')
+                : theme.colors.text.static_icons__tertiary.getVariable('color'),
+        }),
+        selectedDate: {
+            background: `${theme.colors.interactive.link_on_interactive_colors.getVariable(
+                'color'
+            )} !important`,
+            borderColor: `${theme.colors.interactive.primary__resting.getVariable(
+                'color'
+            )} !important`,
```
Comment (eskil0312) [inline src/components/org/PositionTimeline/InlinePositionTimeline/InlineMarker/styles.ts:27]: Avoid use of `!important` if possible

File: src/components/org/PositionTimeline/InlinePositionTimeline/InlineMarker/styles.ts [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+                ? theme.colors.interactive.primary__resting.getVariable('color')
+                : theme.colors.text.static_icons__tertiary.getVariable('color'),
+            background: props.isSelected
+                ? theme.colors.interactive.primary__resting.getVariable('color')
+                : theme.colors.text.static_icons__tertiary.getVariable('color'),
+        }),
+        selectedDate: {
+            background: `${theme.colors.interactive.link_on_interactive_colors.getVariable(
+                'color'
+            )} !important`,
```
Comment (eskil0312) [inline src/components/org/PositionTimeline/InlinePositionTimeline/InlineMarker/styles.ts:24]: Avoid use of `!important` if possible

File: src/components/org/PositionTimeline/InlinePositionTimeline/InlineSplit/components/TooltipContent.tsx [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+};
+
+export const TooltipContent: FC<TooltipContentProps> = ({ split }) => {
+    return (
+        <>
+            <span>
+                {split?.assignedPerson?.name || 'TBN'}
+                <br /> {formatDate(split.appliesFrom)} - {formatDate(split.appliesTo)} (
+                {split.workload || 0}%)
+            </span>
```
Comment (eskil0312) [inline src/components/org/PositionTimeline/InlinePositionTimeline/InlineSplit/components/TooltipContent.tsx:16]: Could we add location in the tooltip?

File: src/components/org/PositionTimeline/components/TimelineSlider/index.tsx [hunk 8]
```diff
@@ -0,0 +1,76 @@
+import { formatDate } from '@equinor/fusion';
```
Comment (eskil0312) [inline src/components/org/PositionTimeline/components/TimelineSlider/index.tsx:1]: Wasn't this file added in the TimlineComponent pr?

Comment (tobiasny) [reply src/components/org/PositionTimeline/components/TimelineSlider/index.tsx:1]: This was a duplicate, and is deleted.

**Review by eskil0312 — APPROVED** [review]

**Pull request merged**